### DISCUSSION TITLE
With Update to 3.5.4 version of codecept Js getting error and execution is not happening

### DISCUSSION THREAD

**Discussion by rohanraviyadav1** [codeceptjs/CodeceptJS#3964] [Q&A]
Error :
**matcher.locator(...).all is not a function**

This error is not coming till 3.5.3 , What change I need to do to resolve this error. Can someone help

**Comment by kobenguyent** [comment]
Please try with latest version and feel free to file an issue with steps to reproduce the issue if this still happens.
